config/settings.yml

```diff
@@ -25,6 +25,10 @@ computacenter:
 cookie_consent:
   expiry_time_months: 6
 
+database_field_encryption:
```

Comment:
Does this actually get used? Seem like it just using the env vars in the code?

Reply:
I hope that these variables get picked up by everyone's `development` environment so that it won't complain about missing environment variables when the code using this eventually goes in. The `test` environment doesn't pick up these variables from `settings.yml` so I had to mock them in the specs. I just thought I'd get it in now before I forgot.